README.md

```diff
@@ -209,7 +211,8 @@ use Zenstruck\Foundry\Proxy;
  * @method static Post[]|Proxy[] randomSet(int $number, array $attributes = []))
  * @method static Post[]|Proxy[] randomRange(int $min, int $max, array $attributes = []))
  * @method static PostRepository|RepositoryProxy repository()
- * @method Post|Proxy create($attributes = [])
+ * @method Post|Proxy create(array|callable $attributes = [])
+ * @method Post[]|Proxy[] many(int $number, array|callable $attributes = [])
```

Comment:
Hmm, I don't think this is right. `many()` returns a `FactoryCollection`. Not sure if `FactoryCollection<Post>|FactoryCollection<Proxy>` would work..?

Reply:
Is there a `Proxy` in the `FactoryCollection`? Isn't it just `Post`?

Reply:
Neither will help PHPStorm and psalm will do well without this. 

I've removed this line.